### PR TITLE
feat: update rpc url for rinkeby and support goerli token registry

### PR DESCRIPTION
**Pull request by isaackps**
## Summary

Enable Goerli title escerow and update rinkeby rpc url

## Changes

- enable goerli title escrow
- update rinkeby rpc url
- update the way we using provider

### CODE REVIEW

File: src/common/config/decrypt.tsx [hunk 1]
```diff
@@ -1,11 +1,12 @@
-import { Wallet, providers } from "ethers";
-import { AwsKmwSignerOption, ConfigFile, ConnectedSigner } from "../../types";
+import { ProviderDetails, utils } from "@govtechsg/oa-verify";
 import { RelayProvider } from "@opengsn/gsn";
```
Comment (superical) [inline src/common/config/decrypt.tsx:2]: Woah.. We have GSN here?🤣

**Review by superical — APPROVED** [review]

File: src/common/config/decrypt.tsx (hunk 1 above)
Comment (isaackps) [reply src/common/config/decrypt.tsx:2]: haha, i think was previously added in but not used

**Comment by superical** [comment]
@isaackps I just notice that the CI test failed for this PR. I haven't looked into it why it timed out but could there be tests in this repo that are using ropsten like some of the other repos?

**Comment by isaackps** [comment]
> @isaackps I just notice that the CI test failed for this PR. I haven't looked into it why it timed out but could there be tests in this repo that are using ropsten like some of the other repos?

yes, there the gsn connected to ropsten. i updated to goerli.

**Comment by superical** [comment]
> > @isaackps I just notice that the CI test failed for this PR. I haven't looked into it why it timed out but could there be tests in this repo that are using ropsten like some of the other repos?
> 
> yes, there the gsn connected to ropsten. i updated to goerli.

Nice, it passes now!😄

**Pull request merged**